internal/archive/tar.go

```diff
@@ -0,0 +1,185 @@
+// Package archive contains functionality to create tar archives
+package archive
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	ignore "github.com/sabhiram/go-gitignore"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/shipa-corp/ketch/internal/errors"
+)
+
+const (
+	shipaIgnoreFile = ".shipaignore"
```

Comment:
ketch is a standalone product. For example, it uses `ketch.yaml` instead of `shipa.yaml`. 
Not sure about `.ketchignore`, it might be that` []string{'.shipaignore', '.ketchignore'}` should be used. Let's discuss it